frontend/src/components/news-page/News.tsx

```diff
@@ -1,3 +1,43 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 
-export const News: React.FunctionComponent = () => <></>;
+export const News: React.FunctionComponent = () => {
+  const [articlesList, setArticlesList] = useState<any[]>([]);
+
+  useEffect(() => {
+    const url = "https://www.reddit.com/r/whales/top.json?limit=50&t=month";
+
+    const fetchData = async () => {
+      try {
+        const response = await fetch(url);
+        const json: any = await response.json();
+        const articles = json.data.children;
+        setArticlesList(
+          articles.filter((article: any) => article.data.is_video)
+        );
+      } catch (error) {
+        console.log("error", error);
+      }
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <>
+      <h1>Whale content</h1>
+      {articlesList &&
+        articlesList.map((article) => (
+          <>
+            <p key={article.data.Id}>
+              {article.data.title.replace(/&amp;/g, "&")}
+            </p>
```

Comment:
I see what you've done here to get around the `&amp;` problem - very clever, but a bit too hacky for my liking! Please have a look at [this StackOverflow answer](https://stackoverflow.com/a/52467292/7629605) - it describes the problem and has two good solutions. One, `dangerouslySetInnerHTML`, is the quickest way of getting this to work:
```suggestion
            <p key={article.data.Id} dangerouslySetInnerHtml={{__html: article.data.title}}>
            </p>
```
but, obviously, that exposes you to XSS (cross-site scripting) attacks, like I told you about yesterday!

![image](https://user-images.githubusercontent.com/4410524/195582234-5d19febc-8feb-4227-95ec-03723db814f8.png)

Instead, please look into the [`html-entities`](https://www.npmjs.com/package/html-entities) solution that the StackOverflow user posted 😊

Thanks guys!